### PR TITLE
Add column name to UnsupportedOperationException for clarity

### PR DESCRIPTION
**Pull request by jwgwalton**
Debugging the valueindexer is quite difficult as it tells you the unsupported type but not what column this is referring to.

### CODE REVIEW

**Comment by imatiach-msft** [comment]
@jwgwalton it looks like the build failed due to style:

##[error]$HOME/agent/_work/1/s/src/value-indexer/src/main/scala/ValueIndexer.scala:76: File line length exceeds 120 characters

could you reduce the line length by splitting it into two lines?

**Comment by mhamilton723** [comment]
also you can run this style check locally with ./runme TESTS="none"

File: src/value-indexer/src/main/scala/ValueIndexer.scala [hunk 1]
```diff
@@ -72,7 +73,7 @@ class ValueIndexer(override val uid: String) extends Estimator[ValueIndexerModel
         case _: DoubleType => sortLevels[JDouble](levels)(NullOrdering[JDouble](Ordering[JDouble]))
         case _: StringType => sortLevels[String](levels)(NullOrdering[String](Ordering[String]))
         case _: BooleanType => sortLevels[JBoolean](levels)(NullOrdering[JBoolean](Ordering[JBoolean]))
-        case _ => throw new UnsupportedOperationException("Unsupported Categorical type " + dataType.toString)
+        case _ => throw new UnsupportedOperationException("Unsupported Categorical type " + dataType.toString + " for column: " + column)
```
Comment (mhamilton723) [inline src/value-indexer/src/main/scala/ValueIndexer.scala]:
```suggestion
        case _ => throw new UnsupportedOperationException(
                "Unsupported Categorical type " + dataType.toString + " for column: " + column)
```

**Comment by imatiach-msft** [comment]
closing PR in favor of https://github.com/Azure/mmlspark/pull/537

**Pull request closed without merging**